### PR TITLE
feat: add http `/health` api

### PR DESCRIPTION
**Pull request by DiamondMofeng**
I hereby agree to the terms of the [GreptimeDB CLA](https://gist.github.com/xtang/6378857777706e568c1949c7578592cc)

## What's changed and what's your intention?

Add `/health` http api for healthy check purpose.
Users are able to visit the api using either `GET` or `POST` methods. Both of them return same response.
The response of this api currently is simply an empty json `{}` with `200 OK` http status code.

## Checklist

- [x]  I have written the necessary rustdoc comments.
- [x]  I have added the necessary unit tests and integration tests.

## Refer to a related PR or issue link (optional)

[#636 ](https://github.com/GreptimeTeam/greptimedb/issues/636)

### CODE REVIEW

**Review by MichaelScofield — APPROVED** [review]
LGTM

File: src/servers/tests/http/http_handler_test.rs [hunk 1]
```diff
@@ -135,3 +135,18 @@ fn create_query() -> Query<http_handler::SqlQuery> {
         database: None,
     })
 }
+
+/// Currently the payload of response should be simply an empty json "{}";
+#[tokio::test]
+async fn test_health() {
```
Comment (waynexia) [inline src/servers/tests/http/http_handler_test.rs:141]: https://github.com/GreptimeTeam/greptimedb/pull/590 add an integration test module that covers serval HTTP APIs, maybe this one can also be included there.

**Review by waynexia — APPROVED** [review]
Looks great! Thank you ❤️ I left a comment about the integration test.

File: src/servers/tests/http/http_handler_test.rs (hunk 1 above)
Comment (DiamondMofeng) [reply src/servers/tests/http/http_handler_test.rs:141]: I will add a test case later. Actually I tried to find where to test http api but missed that part, haha

**Comment by DiamondMofeng** [comment]
wait a minute

**Comment by DiamondMofeng** [comment]
> wait a minute

fixed the format problem

**Pull request merged**